### PR TITLE
Improve measureString performance

### PR DESCRIPTION
**Pull request by marcotisi**
Create only once a <span> element and use it to measure string.
Remove element if Selectize.count equal 0.

### CODE REVIEW

**Comment by marcotisi** [comment]
@joallard this is the PR which improve measureString performances integrated in the main source.

**Comment by marcotisi** [comment]
Any chance to merge this PR in a short time? @joallard

File: src/selectize.js [hunk 1]
```diff
@@ -2049,6 +2049,11 @@ $.extend(Selectize.prototype, {
 		self.$control_input.removeData('grow');
 		self.$input.removeData('selectize');
 
+		if (--Selectize.count == 0 && Selectize.$testInput) {
```
Comment (joallard) [inline src/selectize.js:2052]: What this condition does is not immediatly clear

Comment (joallard) [reply src/selectize.js:2052]: Can you break this out in a better descriptively named function?

**Comment by joallard** [comment]
Do you have benchmark stats?

File: src/selectize.js (hunk 1 above)
Comment (marcotisi) [reply src/selectize.js:2052]: Yeah, I can break into a function and give some benchmarks in a couple of days.

**Pull request merged**